Userland/Libraries/LibGfx/ICCProfile.h

```diff
@@ -47,8 +47,10 @@ char const* device_class_name(DeviceClass);
 
 // ICC v4, 7.2.6 Data colour space field, Table 19 — Data colour space signatures
 enum class ColorSpace : u32 {
-    nCIEXYZ = 0x58595A20,       // 'XYZ '
-    CIELAB = 0x4C616220,        // 'Lab '
+    nCIEXYZ = 0x58595A20, // 'XYZ '
+    PCSXYZ = nCIEXYZ,
+    CIELAB = 0x4C616220, // 'Lab '
```

Comment:
nit: Shouldn't these comments be aligned with the rest of the enum entries, that appears to have been the existing pattern?

Reply:
`clang-format` decided that they shouldn't, when there are entries without comments in between 🤷

Reply:
If you want, I suppose I could add dummy comments to the new lines, but eh.

Reply:
> If you want, I suppose I could add dummy comments to the new lines, but eh.

Yes please. The fact that `clang-format` decided randomly to allow a different style here doesn't seem like a good excuse to not try to have consistent style.

Reply:
Done!